requirements.txt

```diff
@@ -1,2 +1,2 @@
-# web_widget_bokeh_chart
-bokeh==0.12.7
+# generated from manifests external_dependencies
+bokeh
```

Comment:
Should the version be moved to https://github.com/OCA/web/blob/11.0/web_widget_bokeh_chart/__manifest__.py#L17?